module/cmd-alert/src/main/java/net/md_5/bungee/module/cmd/alert/CommandAlertRaw.java

```diff
@@ -23,7 +23,7 @@ public void execute(CommandSender sender, String[] args)
     {
         if ( args.length == 0 )
         {
-            sender.sendMessage( ChatColor.RED + "You must supply a message." );
+            sender.sendMessage( ProxyServer.getInstance().getTranslation( "message_needed" ) );
```

Comment:
I think for completeness it would be better if the default translation for this matches that of send_cmd_usage